assets/timber.scss.liquid

```diff
@@ -1190,6 +1190,79 @@ th, td {
   border: 1px solid $colorBorder;
 }
 
+/*============================================================================
+  Responsive tables, defined with .table--responsive on table element.
+  Only defined for IE9+
+==============================================================================*/
+html:not(.lt-ie9) {
+  .table--responsive {
```

Comment:
If responsive tables are default, we'd have to undo all the styles for older IE. This covers all modern browsers since they won't match the `:not` selector with the least styles defined. Open to suggestions for sure though